.github/workflows/build.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v4
         with:
-          go-version: '1.22'
+          go-version-file: 'go.mod'
```

Comment:
I'm not sure about these changes because they actually decrease the version (we have 1.20 in `go.mod`).

Reply:
That's correct anyway, linter should be set up for the minimum supported version, otherwise it can report some bugs that are relevant for Go 1.22 (like deprecation of some kind), but not relevant Go 1.20.